integration-tests/container/perf/Dockerfile.bpftrace

```diff
@@ -1,4 +1,13 @@
-FROM quay.io/iovisor/bpftrace:latest
+FROM --platform=linux/amd64 quay.io/iovisor/bpftrace:latest AS stage-amd64
+
+FROM --platform=linux/ppc64le ubuntu:focal AS stage-ppc64le
+RUN apt-get update -y && apt-get install -y bpftrace && bpftrace -V
```

Comment:
Looks like the original image also installs `xz-utils`, but either way, if this is all it is, why not just start with a base image and install bpftrace on all arches?